example/common/global_example_menu_component.dart

```diff
@@ -144,7 +144,8 @@ class GlobalExampleMenuComponent extends UiStatefulComponent<
 // ignore: undefined_class
 class GlobalExampleMenuProps extends _$GlobalExampleMenuProps
     // ignore: mixin_of_non_class, undefined_class
-    with _$GlobalExampleMenuPropsAccessorsMixin {
+    with
+        _$GlobalExampleMenuPropsAccessorsMixin {
```

Comment:
Since dartfmt split this across 2 lines, do we need to move the ignore comment down one line?

Reply:
Oh good catch.. I'm actually surprised this didn't cause static analysis to fail. I'll take a look

Reply:
Looks like the example dir wasn't included in analysis, should be fixed now